### ISSUE TITLE
您好，请问该损失函数可以直接应用吗

### ISSUE THREAD

**Issue by SUTwu** [psyrocloud/MS-SSIM_L1_LOSS#3]
您好，请问该损失函数可以直接应用吗，我想使用其替换SSIM损失函数，可是结果好像除了一些问题，损失在无限变大-36789.0508

**Comment by psyrocloud** [comment]
Hello, the loss function can be used directly. We trained stereo and single deblurring networks with it. In your case, please try (1) debugging by training the network with simple L1 loss, (2) Use some pretrained network and test our loss function to see if the loss is absurdly wrong.

**Comment by SUTwu** [comment]
> 您好，损失功能可以直接使用。我们用它训练了立体声和单一的去模糊网络。在你的情况下，请尝试（1）通过用简单的L1损失训练网络来调试，（2）使用一些预先训练的网络并测试我们的损失函数，看看损失是否是错误的。

很高兴看到您的回复，我刚刚尝试用L1损失，确实也出现了离谱的数值。不过我想再问一个问题，我看到您的代码里ms_ssim的比重是0.025，为什么占比这么少呢，个人感觉MS_SSIM的占比应该更高一些

**Comment by psyrocloud** [comment]
> > 您好，损失功能可以直接使用。我们用它训练了立体声和单一的去模糊网络。在你的情况下，请尝试（1）通过用简单的L1损失训练网络来调试，（2）使用一些预先训练的网络并测试我们的损失函数，看看损失是否是错误的。
> 
> 很高兴看到您的回复，我刚刚尝试用L1损失，确实也出现了离谱的数值。不过我想再问一个问题，我看到您的代码里ms_ssim的比重是0.025，为什么占比这么少呢，个人感觉MS_SSIM的占比应该更高一些

这个实现是按照原始文献来的，这个值应该是原作者通过实验得到的较好组合。
